tests/testAutopas/tests/containers/linkedCells/LinkedCellsTest.cpp

```diff
@@ -34,7 +34,7 @@ TYPED_TEST_P(LinkedCellsTest, testUpdateContainer) {
   };
 
   // These are going to be halo particles
-  const std::vector<autopas::Particle> haloParticles{
+  std::vector<autopas::Particle> haloParticles{
       {{-0.5, +1.5, +1.5}, zero, 5},
       {{+5.0, +1.5, +1.5}, zero, 6},
       {{+1.5, -0.5, +1.5}, zero, 7},
```

Comment:
`std::for_each(haloParticles.begin(), haloParticles.end(), [](auto &p) {p.setOwnershipState(autopas::OwnershipState::halo);});`

Instead of setting the ownerships one by one later.

Reply:
I added `OwnershipState` to the particle constructor with default as `owned`. This way I can just add `halo` as the state while creating the particle.